src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1189,6 +1189,116 @@ groovyscript.wiki.naturesaura.spawning.entity.value=Sets the entity spawned
 groovyscript.wiki.naturesaura.spawning.removeByEntity=Removes all Altar of Birthing recipes that summon the given entity
 groovyscript.wiki.naturesaura.spawning.removeByName=Removes the Altar of Birthing recipe with the given name
 
+# ProdigyTech
+groovyscript.wiki.prodigytech.atomic_reshaper.title=Atomic Reshaper
+groovyscript.wiki.prodigytech.atomic_reshaper.description=Uses hot air and Primordium to convert items, can have a weighted random based output
+groovyscript.wiki.prodigytech.atomic_reshaper.time.value=The time needed to perform the recipe, in ticks. By default, takes the value from the ProdigyTech's config
+groovyscript.wiki.prodigytech.atomic_reshaper.primordium.value=The amount of Primordium units consumed by this recipe. 1 Primordium equals 100 Primordium units
+groovyscript.wiki.prodigytech.explosion_furnace.title=Explosion Furnace
+groovyscript.wiki.prodigytech.explosion_furnace.description=Uses an explosive, a dampener, and an optional reagent to convert items, the power values of the recipe, explosive, and dampener should be close to avoid an efficiency loss
+groovyscript.wiki.prodigytech.explosion_furnace.power.value=The power used by this recipe
```

Comment:
since this is used for describing the method, not the field, i think the format of these should be `Sets the ...` instead of `The ...`